website/src/layouts/base/BaseLayout.astro

```diff
@@ -24,6 +24,7 @@ const { title, forceLoggedOutState = false } = Astro.props;
         <meta name='viewport' content='width=device-width' />
         <meta name='generator' content={Astro.generator} />
         <title>{title}</title>
+        <script is:inline defer data-domain='genspectrum.org' src='https://plausible.io/js/script.js'></script>
```

Comment:
Do we have a way to differentiate between staging and prod? Or do we simply not care about that?

Reply:
In Plausible, we can filter by host.